bedrock/products/tests/test_helper_misc.py

```diff
@@ -1011,13 +1011,19 @@ def test_vpn_monthly_price_usd_ca(self):
 
     def test_vpn_monthly_price_euro(self):
         """Should return expected markup"""
-        markup = self._render(plan="monthly", country_code="DE", lang="de")
+        markup = self._render(plan="monthly", country_code="DE", lang="en-US")
```

Comment:
Note for reviewer: updating `lang` here to match the locale of the request used in the test (`en-US`) to make things a bit clearer.